vc2015run.txt

```diff
@@ -0,0 +1,65 @@
+[Section]
+Name = Microsoft Visual C++ 2015 Redistributable Update 3
+Version = 14.0.24212
+License = Unknown
+Description = Needed by applications compiled with Visual Studio 2015. Contains: atl140.dll, concrt140.dll, mfc140.dll, mfc140u.dll, mfcm140.dll, mfcm140u.dll, msvcp140.dll, vcamp140.dll, vccorlib140.dll, vcomp140.dll, vcruntime140.dll.
+Size = 13.7 MiB
```

Comment:
Is `Size` still needed if `SizeInBytes` is present?

Reply:
I think yes, because it always displays the size of the full downloaded file in the program info at the bottom of Rapps window unlike `SizeInBytes` which displays it only in download dialog and only during downloading. Will `Size` value be displayed if I'll remove it from the txt file?

Reply:
if `Size` is still used somewhere, that is a bug that should be fixed.

Reply:
I guess it's a bug in Rapps-db script itself, not from my side.

Reply:
Yes, it's a bug on the rapps side, not yours.
Sorry for the confusion :)